src/tools/pub/transformers/aggregate.md

```diff
@@ -1,128 +0,0 @@
----
```

Comment:
At some point, we might want to forward this to aggregate transformers in build_runner.
https://github.com/dart-lang/build/issues/926#issuecomment-371605646

But that's not ready yet.

I'm unsure that we want to hard-delete this. Aggregate transforming (aggregate building) is really useful. We're just at a point when one way is deprecated and the other way is not well documented yet.

Reply:
Is this an issue if the v1 site is still available?